packages/react-dev-utils/paths/copyPublicFolder.js

```diff
@@ -0,0 +1,7 @@
+var fs = require('fs-extra');
+module.exports = paths => function copyPublicFolder() {
```

Comment:
This one's here cause it can be reused by other scripts, like watch for example :)